pyne/partisn.py

```diff
@@ -586,7 +561,7 @@ def _write_input(title, block01, block02, block03, block04, block05, name=None):
         for num in row:
             partisn += "{} ".format(num)
             count += 1
-            if count == 20:
+            if count == 10:
```

Comment:
The `wrap(s, 80)` is a pretty big job because it really should be done on the whole file. I think that should wait.